mds/access_control/stateless_jwt.py

```diff
@@ -0,0 +1,31 @@
+from django.utils.encoding import smart_text
+from rest_framework.authentication import BaseAuthentication, get_authorization_header
+
+from mds.server.settings import AUTH_MEANS
```

Comment:
`sever.settings` is an example server application, using it directly prevents usage of `mds.access_control/stateless_jwt` in another application using `django-mds` as a library.
Use `from django.conf import settings` to use this setting agnostically.

Reply:
thans !